core/configuredivecomputerthreads.cpp

```diff
@@ -2072,7 +2072,7 @@ static dc_status_t write_ostc_settings(dc_device_t *device, DeviceDetails *m_dev
 	//sync date and time
 	if (m_deviceDetails->syncTime) {
 		QDateTime timeToSet = QDateTime::currentDateTime();
-		dc_datetime_t time = { 0 };
+		dc_datetime_t time = { 0,0,0,0,0,0,0 };
```

Comment:
at some point we had a compiler who didn't like this, but Travis passes, so this might be history.